pkg/services/ngalert/schedule/schedule.go

```diff
@@ -340,15 +342,15 @@ func (sch *schedule) ruleEvaluationLoop(ctx context.Context) error {
 
 			type readyToRunItem struct {
 				key      models.AlertRuleKey
-				ruleInfo alertRuleInfo
+				ruleInfo *alertRuleInfo
```

Comment:
We are changing this from a `alertRuleInfo` to a `*alertRuleInfo` because it now holds a mutex right?

Reply:
Correct